src/modules/projects/components/ProjectOutgoingReferralDetailsSubForm.tsx

```diff
@@ -57,11 +61,24 @@ const ProjectOutgoingReferralDetailsSubForm: React.FC<Props> = ({
     );
   }
 
+  const { data: receivingProject } = useGetCanProjectAcceptReferralQuery({
```

Comment:
feels a little funny to call this `receivingProject` when it's just a data object with a boolean. I think it would be clearer to just leave it as `data` or destructure the boolean field directly

Reply:
I'll leave it as `data`, that way we can also check whether data has been returned yet or not, so the alert doesn't show up when the result hasn't been returned yet.